server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1444,6 +1426,55 @@ public void run() {
     log.info("exiting");
   }
 
+  class ReplTServer {
```

Comment:
What value is this class giving us? Just seems like an extra object we create.

Reply:
This was so we can pass a final object to the timer thread.

Reply:
What about just using `AtomicReference` instead of defining our own class?

Reply:
Oh yeah good idea, that would be better.